client/modules/IDE/components/FileNode.jsx

```diff
@@ -11,6 +11,7 @@ import DownArrowIcon from '../../../images/down-filled-triangle.svg';
 import FolderRightIcon from '../../../images/triangle-arrow-right.svg';
 import FolderDownIcon from '../../../images/triangle-arrow-down.svg';
 import FileTypeIcon from './FileTypeIcon';
+import { sizeLimit, currentSize } from './AssetSize';
```

Comment:
This doesn't seem to import the correct constants you've created in `AssetSize.jsx`!